chef/cookbooks/nova/recipes/compute.rb

```diff
@@ -106,6 +106,10 @@ def set_boot_kernel_and_trigger_reboot(flavor='default')
 end
 
 case node[:nova][:libvirt_type]
+  when "zvm"
+    package "openstack-nova-zvm"
```

Comment:
okay, i will rename it.
@dirkmueller how shall we call the neutron package? any ideas?